park_info.clj

```diff
@@ -1,198 +1,197 @@
-(
-    def Bertramka{
-        ;---Basic Info---
-        :food nil
-        :food_extra nil
-        :toilet true
-        :toilet_extra nil
-        :POIs "Cultural monument, classical music concerts, social events, W.A. Mozart Museum" ;POI refers to Point of Interest
-        ;---Children and Sport---
-        :bike_path true
-        :skating false
-        :sports_field false
-        :sports_field_extra: nil
-        :playground false
-        :playground_extra: nil
-        ;---Directions---
-        :public_transport "
-        stop: Bertramka - trams: #4, #7, #9, #10, #58, #59"
-        :GPS_coords "50°4′15.00″ N, 14°23′42.00″ E"
-        :parking true
-        :parking_extra "OC Smíchov underground parking"
-        ;---Additional Information---
-        :dogs nil
-        :path_type "paved"
-        :administered_by "Prague 5 & Comenius Company"
-        :opening_hours "April-October: 09:00-18:00, November-March: 09:30-16:00"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/fe/bd/6a/184189_4_bertramka_mapa.pdf"
+(def Bertramka{
+    ;---Basic Info---
+    :food nil
+    :food_extra nil
+    :toilet true
+    :toilet_extra nil
+    :POIs "Cultural monument, classical music concerts, social events, W.A. Mozart Museum" ;POI refers to Point of Interest
```

Comment:
No 'suitable for'?

Reply:
These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.